src/styles/normalize.scss

```diff
@@ -0,0 +1,23 @@
+body {
+  margin: 0;
+  font-family: Roboto, Helvetica, sans-serif;
+  font-weight: 500;
+}
+
+header,
+a,
+img,
+nav,
+ul,
+li,
+div,
+p,
+h1,
+h2,
+h3,
+h4,
+h5 {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
Don't reset styles for tags that don't need it. For example, the `img` tag does not have `padding` or `margin` by default, so there is no need to reset it. You can see more detailed information about tags and their standard styles here:
https://css.in.ua/html/tag/img

Also, don't reset styles for tags you don't use in your code. For example, I don't see you using tags `h1`, `h2`, `h4`, `h5`.